app/views/rulesets/index.html.erb

```diff
@@ -11,8 +11,14 @@
             </h1>
           </legend>
           <div class="govuk-radios" data-module="govuk-radios">
+            <% Linters::Rulesets::ACTIVE_RULESETS.each do |ruleset_name| %>
+              <div class="govuk-radios__item">
+                <%= text_field_tag("ruleset_name", ruleset_name, class: "govuk-radios__input", type: "radio", name: "ruleset_name", checked: true) %>
+                <%= label_tag("gov-ruleset", t("rulesets.#{ruleset_name}.description"), class: "govuk-label govuk-radios__label", for: "gov-ruleset") %>
```

Comment:
[change]

The [`for` field should point at the id of the radio button](https://stackoverflow.com/questions/1527721/using-label-for-on-radio-buttons) element the label applies to. Looking at the radio code, I think that will be `"ruleset_name_#{ruleset_name}"`. I can't see anything on the page that "gov-ruleset" will match.

Reply:
This is now done as per Rob's suggestion.

Reply:
To clarify. Line 17 should be I think:
```ruby
<%= label_tag("gov-ruleset", t("rulesets.#{ruleset_name}.description"), class: "govuk-label govuk-radios__label", for: "ruleset_name_#{ruleset_name}") %>
```
Though worth firing the app up and checking the id of the radio button input to make sure I've constructed it correcly